src/Microsoft.AspNetCore.Rewrite/Internal/CodeRules/RedirectRule.cs

```diff
@@ -54,7 +56,7 @@ public override void ApplyRule(RewriteContext context)
 
                 if (string.IsNullOrEmpty(newPath))
                 {
-                    response.Headers[HeaderNames.Location] = "/";
+                    response.Headers[HeaderNames.Location] = pathBase + '/';
```

Comment:
Believe so. The pathbase can be null/ empty, so we need to guarantee there is a slash there.